Source/WebKit/WebProcess/InjectedBundle/DOM/InjectedBundleRangeHandle.cpp

```diff
@@ -175,7 +175,8 @@ RefPtr<WebImage> InjectedBundleRangeHandle::renderedImage(SnapshotOptions option
 String InjectedBundleRangeHandle::text() const
 {
     auto range = makeSimpleRange(m_range);
-    range.start.document().updateLayout();
+    Ref document = range.start.document();
```

Comment:
Why not `range.start.protectedDocument()->updateLayout()`?

Reply:
Done.